src/navigation-breadcrumbs.tsx

```diff
@@ -1,35 +1,88 @@
+/**
+ * Figma: https://www.figma.com/file/BctaFHf0PVPX2uwRSl8taf/Breadcrumbs?node-id=2%3A61
+ * Accessibility: https://www.w3.org/WAI/ARIA/apg/example-index/breadcrumb/index.html
+ */
 import * as React from 'react';
 import {Text1} from './text';
 import {useTheme} from './hooks';
+import {createUseStyles} from './jss';
+import {DataAttributes} from './utils/types';
+import {getPrefixedDataAttributes} from './utils/dom';
 
 const BREADCRUMB_SEPARATOR = ' / ';
 
+const useStyles = createUseStyles(() => ({
+    link: {
+        '&:hover': {
+            textDecoration: 'underline',
+        },
+    },
+    current: {
+        textDecoration: 'none',
+        pointerEvents: 'none',
+    },
+    list: {
+        padding: 0,
+        margin: 0,
+        listStyleType: 'none',
+        '& > li': {
+            display: 'inline',
+        },
+    },
+}));
+
 export type NavigationBreadcrumbsProps = {
     title: string;
     breadcrumbs: ReadonlyArray<{
         readonly title: string;
         readonly url: string;
     }>;
     children?: void;
+    dataAttributes?: DataAttributes;
+    'aria-label'?: string;
 };
 
-const NavigationBreadcrumbs: React.FC<NavigationBreadcrumbsProps> = ({title, breadcrumbs}) => {
+const NavigationBreadcrumbs: React.FC<NavigationBreadcrumbsProps> = ({
+    title,
+    breadcrumbs,
+    dataAttributes,
+    'aria-label': ariaLabel = 'Breadcrumb',
+}) => {
     const theme = useTheme();
     const Link = theme.Link;
+    const classes = useStyles();
     return (
-        <div>
-            {breadcrumbs.map(({title, url}, index) => (
-                <React.Fragment key={index}>
-                    <Link style={{textDecoration: 'none'}} to={url}>
-                        <Text1 regular>{title}</Text1>
-                    </Link>
-                    <Text1 regular>{BREADCRUMB_SEPARATOR}</Text1>
-                </React.Fragment>
-            ))}
-            <Text1 regular color={theme.colors.textSecondary}>
-                {title}
-            </Text1>
-        </div>
+        <nav aria-label={ariaLabel} {...getPrefixedDataAttributes(dataAttributes)}>
+            <ol className={classes.list}>
+                {breadcrumbs.map(({title, url}, index) => (
+                    <li key={index}>
+                        <Link style={{textDecoration: 'none'}} to={url}>
+                            <span className={classes.link}>
+                                <Text1 regular>{title}</Text1>
```

Comment:
maybe you can use TextLink to have the same behavior in inverse background?
https://ibit.ly/kEVw

(I saw specs and this behavior is not contemplate because the change in textLink was later, we will update specs to show inverse behavior)

Reply:
https://github.com/Telefonica/mistica-web/pull/520

TextLink in inverse context have to show underline and on hover increase underline stroke to 2px

Reply:
ok, done